modules/access/variables.tf

```
@@ -224,3 +224,9 @@ variable "factory_monitoring" {
   type        = bool
   default     = true
 }
+
+variable "additional_target_groups" {
```

nit: let's make this `access_target_group_arns` so that it's easier to search the repo and see how the top-level variable is propagated to the submodules
```suggestion
variable "access_target_group_arns" {
```

---

@chrnorm I've incorporated the suggested changes; good call on searchability. Tested against my own deployment post-change.